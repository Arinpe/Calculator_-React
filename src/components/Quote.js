const Quote = () => (
  <p className="quote">
    “Common sense is calculation applied to life. ”
    ― Henri Frederic Amiel
  </p>
);
export default Quote;
