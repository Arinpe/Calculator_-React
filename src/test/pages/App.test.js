import React from 'react';
import render from 'react-test-renderer';
import Quote from '../../components/App';

describe('Quote', () => {
  const page = render.create(<Quote />);
  const pageInstance = page.root;
  test('should render without crashing', () => {
    expect(page).toMatchSnapshot();
    expect(pageInstance.findAllByProps({ className: 'nav-links' }).length).toEqual(1);
    expect(pageInstance.findAllByProps({ className: 'logo' }).length).toEqual(1);
  });
});
