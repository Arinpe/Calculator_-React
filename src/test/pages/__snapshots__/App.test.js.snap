// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should render without crashing 1`] = `
Array [
  <div
    className="nav"
  >
    <h1
      className="logo"
    >
      Math Magicians
    </h1>
    <div
      className="nav-links"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="nav-link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        className="nav-link"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </div>,
  <div
    className="home"
  >
    <h1>
      Welcome to our page!
    </h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia facilis quo, deleniti at debitis commodi neque eaque dolore! Dolorem voluptatum obcaecati ipsum ratione voluptatibus consequuntur dolor sequi magnam molestiae voluptas nisi iure hic, cumque officia asperiores aspernatur dolores fugiat pariatur consectetur reiciendis quis voluptatem dicta nemo rem. Dolor nobis quia magnam libero ea, impedit mollitia voluptas dolores maxime? Recusandae ratione quas molestiae cupiditate placeat deserunt temporibus velit suscipit pariatur laborum. Veritatis, et ratione accusantium delectus repellat cupiditate voluptas tempore laboriosam amet facere rem omnis perferendis veniam magni dignissimos hic consectetur dolorum aliquid non est qui provident ex? Iusto, eos delectus!
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi quas autem harum voluptatibus officia doloremque? Consectetur aut suscipit quas excepturi nisi consequuntur distinctio explicabo sunt, voluptatum rerum eaque quisquam provident inventore assumenda impedit. Quam saepe delectus voluptate tenetur veniam repudiandae similique ipsum eius aspernatur, ut dolorum cupiditate maxime, ratione, voluptatibus ab nam dolorem temporibus expedita deserunt quae. Numquam, expedita autem commodi fugit necessitatibus repudiandae. Deserunt iure accusantium dolorum aliquam voluptatum. Ipsam aperiam aspernatur numquam corporis culpa doloribus commodi ex impedit.
    </p>
  </div>,
]
`;
